### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<html>
<body>

Site | URLScan link
-- | --
serverfixnow.online|https://urlscan.io/result/7a57fb1d-d0d4-4d37-9a5a-9206897f6dae/dom/
myconnectwalletdapp.online|https://urlscan.io/result/ea238bbf-5603-4d74-b5a6-1b119e1d8096/dom/

</body>
</html>

		"ismetamask.io",
		"metamask-verify-procedure.com",
		"metamask.io.wallet-identification.softsayo.com",
		"metamask.recovery-web.io",
		"recovery-web.io",
		"metamask02.com",
		"metamasksecure.org",
		"myconnectwalletdapp.online",
		"serverfixnow.online",

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -14,6 +14,16 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"bitgert.swap.defi-token.my.id",
+		"bridge.hertzswap.net",
```
Comment (jacogr) [inline all.json:18]: Nothing here ... yet. However `hertzswap.net` does seem like it connects, i.e. doesn't ask for seeds.

File: all.json [hunk 2]
```diff
@@ -14,6 +14,16 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"bitgert.swap.defi-token.my.id",
```
Comment (jacogr) [inline all.json:17]: Should add `defi-token.my.id` as well?

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json:18]: Thanks for your vigilance, coffee has not kicked in yet 😴

**Review by dubstard — COMMENTED** [review]
remove `bridge.hertzswap.net` (no bad content...yet)

add more scams, same IP

**Review by dubstard — COMMENTED** [review]
add "defi-token.my.id",

File: all.json (hunk 2 above)
Comment (dubstard) [reply all.json:22]: Yup yup!

**Review by dubstard — COMMENTED** [review]
remove duplicate  "babydogenfft.online",

**Comment by dubstard** [comment]
✅Good to go

**Review by jacogr — APPROVED** [review]

**Comment by dubstard** [comment]
Thanks!

**Pull request merged**